Home/index.html

```diff
@@ -47,18 +55,192 @@
+                        dummy text.
+                    </p>
+                    <ul>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="content">
+                    <h1>Website Portfolio</h1>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. has been the industry's standard dummy text
+                        ever since the 1500s, when an unknown printer took a standard
+                        dummy text.
+                    </p>
+                    <ul>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section id="about" class="about">
+        <div class="about_text">
+            <h1>About me</h1>
+            <p>
+                Hello I'm a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded, don't hesitate to
+                contact me.
+            </p>
+            <a href="#">Get My Resume</a>
+        </div>
+        <div class="line"></div>
+        <div class="skills_container">
+            <div class="language">
+                <div>
+                    <div class="deco1"></div>
+                    <h4>
+                        Language
+                    </h4>
+                </div>
+                <ul>
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+            <div class="frameworks">
+                <div>
+                    <div class="deco2"></div>
+                    <h4>
+                        Frameworks
+                    </h4>
+                </div>
+                <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+            <div class="skills">
+                <div>
+                    <div class="deco3"></div>
+                    <h4>
+                        Skills
+                    </h4>
+                </div>
+                <ul>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
```

Comment:
Good job for using ul/li for listing the specificity like Codekit, Codepen ... 👍 

For the same reason, you have to use ul/li for listing these 3 elements : Languages ,Frameworks ,Skills